packages/spor-react/src/input/CardSelect.tsx

```diff
@@ -129,8 +132,10 @@ export const CardSelect = forwardRef<CardSelectProps, "button">(
             offset={size === "sm" ? 6 : 12}
             crossOffset={crossOffset}
             placement={placement}
+            
           >
             <Card
+              backgroundScheme={backgroundScheme}
```

Comment:
Hva skjer når man sender inn `backgroundScheme` til cards? Tror ikke den støtter det.

Altså så burde vi kanskje tenke på en annen navngivning enn backgroundScheme. Dark og green er litt uheldige navn med tanke på multibrand (kanskje bedre med primary og secondary, eller base og accent) eller noe. Og vil man noensinne trenge begge versjoner på én side? Hvis ikke burde vi kanskje løse det på toppnivå, heller enn her nede.

Reply:
![Screenshot 2023-12-20 at 16 35 20](https://github.com/nsbno/spor/assets/69791398/8ace3cfb-864e-49a6-b3e9-fc057f35d2ab)

tenker du når man sender det inn her eller etter annet sted? 

enig med at det kan være et bedre navn enn "backgroundScheme" kom bare ikke på noe bra :/

endret dark og green til primary og secondary. base og accent blir blandet med variants. jeg fulgte bare per sin kommentar på forrige pr om card select at han ville ha det som en prop der man endre on the go (kan hende jeg misforsto han)